sql-parser/src/test/java/io/crate/sql/parser/TestSqlParser.java

```diff
@@ -401,13 +396,6 @@ public void testTrimFunctionMissingFromWhenCharsToTrimIsPresentThrowsException()
         assertInstanceOf("TRIM(' ' chars)", FunctionCall.class);
     }
 
-    @Test
-    public void testTrimFunctionInvalidTrimModeThrowsException() {
-        expectedException.expect(ParsingException.class);
-        expectedException.expectMessage("line 1:12: no viable alternative at input 'TRIM(BOTHH ' '");
-        assertInstanceOf("TRIM(BOTHH ' ' FROM chars)", FunctionCall.class);
-    }
```

Comment:
Removed as it's a no longer an invalid test. 

`BOTHH ' '` results in a cast expression which will later fail (`Cannot find data type: bothh`).
However, for this context, it is valid - thus It's removed completely.